src/lexers/PHPLexer.js

```diff
@@ -51,6 +51,10 @@ define(function (require, exports, module) {
             })
             // ignore the comments.
             .addRule(/\/\/[^\n]*/, ignored)
+            // ignore strings (double quotes).
+            .addRule(/"((?:\\.|[^"\\])*)"/, ignored)
+            // ignore strings (single quotes).
+            .addRule(/'((?:\\.|[^'\\])*)'/, ignored)
```

Comment:
Should this also add backticks?
https://secure.php.net/manual/en/language.operators.execution.php